server/planning/planning/planning_post.py

```diff
@@ -113,3 +122,21 @@ def _get_post_state(self, plan, new_post_state):
             return WORKFLOW_STATE.SCHEDULED
 
         return plan.get('state')
+
+    @staticmethod
+    @celery.task(soft_time_limit=600)
+    def _enqueue_planning(id):
```

Comment:
`_enqueue_planning` and `_enqueue_event`  can be just one method `enqueue_item`. There is `type` store in the collection.

Reply:
I'll move it to common.py